pantab/tests/test_roundtrip.py

```diff
@@ -69,8 +68,72 @@ def test_roundtrip_with_external_hyper_process(df, tmp_hyper):
     with HyperProcess(
         Telemetry.DO_NOT_SEND_USAGE_DATA_TO_TABLEAU, parameters=parameters
     ) as hyper:
+        # test frame_to_hyper/frame_from_hyper
         pantab.frame_to_hyper(df, tmp_hyper, table="test", hyper_process=hyper)
-        pantab.frame_from_hyper(tmp_hyper, table="test", hyper_process=hyper)
-        pantab.frames_from_hyper(tmp_hyper, hyper_process=hyper)
+        result = pantab.frame_from_hyper(tmp_hyper, table="test", hyper_process=hyper)
+        assert_roundtrip_equal(result, df)
+
+        # test frame_from_hyper_query
+        result = pantab.frame_from_hyper_query(
+            tmp_hyper, "SELECT * FROM test", hyper_process=hyper
+        )
+        assert result.size == 63
+
+        # test frames_to_hyper/frames_from_hyper
+        pantab.frames_to_hyper(
+            {"test2": df, "test": df}, tmp_hyper, hyper_process=hyper
```

Comment:
note how the semantics differs between `frames_to_hyper(..., connection)` and `frames_to_hyper(..., path)`:
The former keeps already existing tables, the later one removes them. Which behavior do we prefer?

Reply:
@WillAyd

Reply:
Yea we don't have very clear semantics on this yet. There probably are different use cases for both, but I think the solution to that problem is orthogonal to this PR

Reply:
removed the functionality for writing again...
we can reintroduce this later, after figuring out a clear semantics